feature/gnoi/os/tests/osinstall/osinstall_test.go

```diff
@@ -88,9 +89,14 @@ func TestOSInstall(t *testing.T) {
 	tc.fetchStandbySupervisorStatus(ctx, t)
 	tc.transferOS(ctx, t, false)
 	tc.activateOS(ctx, t, false)
-	if tc.dualSup {
-		tc.transferOS(ctx, t, true)
-		tc.activateOS(ctx, t, true)
+	switch dp.Device().Vendor() {
+	case ondatra.ARISTA, ondatra.CISCO:
```

Comment:
For Cisco we also do not need transfer and activate for standby rp, please move the cisco to the next case.